gui-backend/src/main/java/com/rln/gui/backend/implementation/GuiBackendApiImplementation.java

```diff
@@ -134,96 +114,113 @@ public List<LedgerAddressDTO> get() {
     return autoApproveApi.get();
   }
 
-  // * what is a client?
-  // Get all the clients
+  // * What is a local address? (in our model) do we know whether something is local?
+  // Get the balance for an address, provided the address is local to this participant
   @Override
-  public List<ClientDTO> get1() {
+  public List<Balance> getLocalBalance(String address) {
+    try {
+      return balancesApi.getLocalBalance(address);
+    } catch (IbanNotFoundException e) {
+      throw notFound();
+    }
+  }
+
+  // Wallet management ----------
+
+  // Create a new wallet address
+  @Override
+  public void post3(Long walletId, @Valid @NotNull WalletAddressDTO walletAddressDTO) {
     throw notImplemented();
   }
 
-  // * what kind of party is this about?
-  // Get all the known parties
+  // Create a new wallet
   @Override
-  public List<PartyDTO> get2() {
+  public void postWallets(@Valid @NotNull WalletDTO walletDTO) {
     throw notImplemented();
   }
 
-  // * clarification about wallets and addresses
-  // Get all the wallet addresses in the specified wallet
+  // Get all the wallets
```

Comment:
Are these supposed to be Javadocs? I see these come from swagger Yaml. Can we configure the generator to make turn these into actual Javadocs? 🤔